src/js/app.js

```diff
@@ -79,18 +69,13 @@ injectTapEventPlugin();
 // a <script> tag. Also, a <div class="main" /> is required in the body. We
 // render the application into that div.
 document.addEventListener( 'deviceReady', ( ) => {
-  const storeBuilder = new StoreBuilder( [ gateway.getMiddleware() ] );
-  const store = storeBuilder.build();
-
-  const network = new NetworkManager( store );
-  network.monitor();
+  const network = new NetworkStateAgent( store );
+  network.run();
 
-  const sync = new Sync( database, gateway, store );
-  sync.start();
+  const replicator = new ReplicationAgent( store );
+  replicator.run();
```

Comment:
`NetworkStateAgent` and `ReplicationAgent` replace `NetworkManager` and `Sync`. Both are "agents" which subscribe to store updates and react to some change.